### PR TITLE
Updates to documentation while reviewing translations (smoe:Docs29 mac 102)

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

File: docs/man/man3/hal_add_funct_to_thread.3hal [hunk 1]
```diff
@@ -32,7 +32,7 @@ This determines how often and in what order functions are executed.
 Returns a HAL status code.
 
 .SH REALTIME CONSIDERATIONS
-Call only from realtime init code, not from realtime or non-realtime code.
+Call only from realtime init code, not from anderem realtime or non-realtime code.
```
Comment (SebKuzminsky) [inline docs/man/man3/hal_add_funct_to_thread.3hal:35]: Oops, wrong language!  ;-)

File: docs/man/man3/hal_create_thread.3hal [hunk 2]
```diff
@@ -18,23 +18,21 @@ The name of the thread
...
-All thread periods are rounded to integer multiples of the hardware timer
-period, and the timer period is based on the first thread created.  Threads
-must be created in order, from the fastest to the slowest.  HAL assigns
-decreasing priorities to threads that are created later, so creating them
-from fastest to slowest results in rate monotonic priority scheduling.
+All thread periods are rounded to integer multiples of the hardware timer period,
+and the timer period is based on the first thread created.
+Threads must be created in order, from the fastest to the slowest.
+HAL assigns decreasing priorities to threads that are created later,
+so creating them from fastest to slowest results in rate monotonic priority scheduling.
```
Comment (SebKuzminsky) [inline docs/man/man3/hal_create_thread.3hal:30]: This hunk looks different from what I expect - usually "semantic line-breaks" put each sentence on its own line.  These sentences are still split across multiple lines.  This pattern happens in a couple of places in this PR.

Or is something else going on here that I don't understand?

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]

File: docs/man/man3/hal_create_thread.3hal (hunk 2 above)
Comment (smoe) [reply docs/man/man3/hal_create_thread.3hal:30]: One sentence per line is the best possible. Here, I was just using the comma as a natural location for a line break, which typically (but not always, depends on the the kind of the comma) is enough for the auto-translators not to mess it up. Had felt a bit bad to have a line that is twice as long as the other lines in this man page.

File: docs/man/man3/hal_add_funct_to_thread.3hal (hunk 1 above)
Comment (smoe) [reply docs/man/man3/hal_add_funct_to_thread.3hal:35]: ;) Darn.

File: docs/man/man3/hal_create_thread.3hal (hunk 2 above)
Comment (SebKuzminsky) [reply docs/man/man3/hal_create_thread.3hal:30]: Ok :-)

**Comment by SebKuzminsky** [comment]
Looks like this PR branch needs a rebase, then let's merge it.

File: docs/man/man1/milltask.1 [hunk 3]
```diff
@@ -3,22 +3,16 @@
...
-\fBmilltask\fR is an internal process of LinuxCNC.  It is generally not
-invoked directly but by an INI file setting: \fB[TASK]TASK=milltask\fR.
-The \fBmilltask\fR process creates the \fBini.*\fR HAL pins listed below
-and owned by the \fBinihal\fR component.  These pins may be modified
-while LinuxCNC is running to alter values that are typically specified
-in an INI file.
+\fBmilltask\fR is an internal process of LinuxCNC.
+It is generally not invoked directly but by an INI file setting: \fB[TASK]TASK=milltask\fR.
+The \fBmilltask\fR process creates the \fBini.*\fR HAL pins listed below and owned by the \fBinihal\fR component.
+These pins may be modified while LinuxCNC is running to alter values that are otherwise statically specified in an INI file.
```
Comment (hansu) [inline docs/man/man1/milltask.1:9]:
```suggestion
These pins may be modified while LinuxCNC is running to alter values that are typically statically specified in an INI file.
```

I find the original wording more suitable

File: docs/src/gui/gladevcp.adoc [hunk 4]
```diff
@@ -220,7 +220,7 @@ This will look like this:
 image::images/glade-manual-small.png[]
 
 Glade tends to write a lot of messages to the shell window, which mostly can be ignored.
-Select 'File'→'Save as', give it a name like 'myui.ui' and make sure it's saved as 'GtkBuilder' file (radio button left bottom corner in Save dialog).
+Select 'File'→'Save as', give it a name like 'myui.ui' and make sure it is saved as 'GtkBuilder' (radio button left bottom corner in Save dialog).
```
Comment (hansu) [inline docs/src/gui/gladevcp.adoc:223]:
```suggestion
Select 'File'→'Save as', give it a name like 'myui.ui' and make sure it is saved as 'GtkBuilder' file (radio button left bottom corner in Save dialog).
```
I prefer the version with 'file'

File: docs/src/gui/gladevcp.adoc [hunk 5]
```diff
@@ -2117,16 +2117,15 @@ A simplex example is the `ESTOP` toggle button.
 
 Currently the following widgets are available:
 
-- The `ESTOP` toggle sends `ESTOP` or `ESTOP_RESET` commands to LinuxCNC depending on it's state.
+<<<<<<< HEAD
```
Comment (hansu) [inline docs/src/gui/gladevcp.adoc:2120]: Here something went wrong

**Review by hansu — CHANGES_REQUESTED** [review]

**Review by hansu — APPROVED** [review]

**Pull request merged**